src/AppInstallerCLICore/Command.cpp

```diff
@@ -26,11 +26,12 @@ namespace AppInstaller::CLI
 
     Command::Command(
         std::string_view name,
+        std::vector<std::string_view> aliases,
         std::string_view parent,
         Command::Visibility visibility,
         Settings::ExperimentalFeature::Feature feature,
         Settings::TogglePolicy::Policy groupPolicy) :
-        m_name(name), m_visibility(visibility), m_feature(feature), m_groupPolicy(groupPolicy)
+        m_name(name), m_aliases(aliases), m_visibility(visibility), m_feature(feature), m_groupPolicy(groupPolicy)

```

Comment:
```suggestion
        m_name(name), m_aliases(std::move(aliases)), m_visibility(visibility), m_feature(feature), m_groupPolicy(groupPolicy)
```
This solves @lechacon 's comment above.